core/src/it/scala/org/ensime/fixture/SearchServiceFixture.scala

```diff
@@ -12,9 +12,11 @@ import org.ensime.AkkaBackCompat
 import org.ensime.indexer.SearchService
 import org.ensime.vfs._
 
+
 trait IsolatedSearchServiceFixture extends IsolatedSourceResolverFixture with AkkaBackCompat {
 
   def withSearchService(testCode: (EnsimeConfig, SearchService) => Any)(implicit actorSystem: ActorSystem, vfs: EnsimeVFS): Any = withSourceResolver { (config, resolver) =>
+    import EnsimeConfigFixture.serverConfig
```

Comment:
Where to import ? The top level import(as in previous commit wasnt helping). Import in this commit is not working either.